### ISSUE TITLE
Feature Proposal: Save bonuses increase by level by class

### ISSUE THREAD

**Issue by Turtlelord26** [trowl223/Compound_X#718]
To allow easier balance of Saves across the leveling experience, to differentiate the classes more, and to help out #382, we should consider Save bonus scaling.

Characters would receive a +X, +Y, +Z, +W to their saves each level, and we could apportion fast/med/slow growth to each save differently based on class, to mirror the kind of training that class makes use of.

Must keep #383 in mind, as work here will affect what kind of work can be done there.

**Comment by Turtlelord26** [comment]
Here's a draft of some numbers.
![image](https://user-images.githubusercontent.com/23040366/54254226-3f947d00-4510-11e9-8bf8-dd032ed52ee2.png)

**Comment by Turtlelord26** [comment]
Slow, medium, and fast are a floor function on level * (1.5, 2.5, or 3.5).
Rates are given to classes as a compromise between flavor and an even distribution of highs and lows. Naturally, subject to feedback.

**Comment by Turtlelord26** [comment]
To clarify, this is intended to be in addition to the current numbers generated by characters' primary stats. A lv 1 specialist with 6 int and 6 cha would have a Will save of 12 + 2 = 14.

**Comment by bleehu** [comment]
At level 5, with a fast progression, you've doubled the average save that a character would make. That's not an insurmountable problem, but it does mean that I'd have to re-balance all of the DCs in the enemy bestiary. 
The call-out of the confluence of this with 383 makes me think that this is a small feature that we may want to make a project out of?

**Comment by Turtlelord26** [comment]
Yeah, I could see this existing as a small project. Would be nice and organized that way. Before we go there though, I'd like at least a short conversation on those save numbers and how we want defensive ranges and offensive thresholds to line up. No rush on this project, and I think it would help to make sure we're all starting from the same spot.
